.gitignore

```diff
@@ -36,3 +36,6 @@
 
 # Ignore node_modules
 node_modules
+
+# Ignore RubyMine
+.idea/*
```

Comment:
@srb- are you using rubymine for this projecT?

Reply:
Yeah, just testing it out for fun. But I'll only seriously consider it if it works for Hangar.